### PR TITLE
Create a GET product with id method

### PR DESCRIPTION
**Pull request by pinocchiochang**
create api to GET product with product ID as parameter, so that to get the information of the specific item.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -38,6 +38,28 @@ def get_product_list():
...
+  """ Get info about a specific product
+
+    This method will get the info about an item with it's product id
+
+    Args:
+      id (int): The id of the product to be update
+
+    Returns:
+      response: product id information(product id, location id, used/new/open_box, total_quantity, restock_level)
+      status 200 if succeeded
```
Comment (omersolmazer) [inline server.py:53]: What if the product is not found?

**Review by omersolmazer — COMMENTED** [review]
Which status is returned if the product id is not registered in the database?

File: server.py (hunk 1 above)
Comment (pinocchiochang) [reply server.py:53]: I missed that.

**Review by p2hang — APPROVED** [review]
Looks good!

File: server.py (hunk 1 above)
Comment (omersolmazer) [reply server.py:53]: Thanks for update!

**Pull request merged**